components/BottomNavigation/examples/MDCBottomNavigationBarController.m

```diff
@@ -38,6 +38,17 @@ - (instancetype)init {
   return self;
 }
 
+- (instancetype)initWithCoder:(NSCoder *)aDecoder {
+  self = [super initWithCoder:aDecoder];
+  if (self) {
+    _navigationBar = [[MDCBottomNavigationBar alloc] init];
+    _content = [[UIView alloc] init];
+    _viewControllers = @[];
+    _selectedIndex = NSNotFound;
+  }
+  return self;
+}
```

Comment:
Please pull the common init logic to a shared initializer.

Reply:
Done